<details><summary>Failed to submit graph for 9e99a628148d14401ddd285aeea9da4ec4096455
</summary>Message:
Failed to run command: 'pbp generate from-feedstock --output /tmp/tmpfyartebe/tensorflow-feedstock.yaml --custom-metadata-file /tmp/tmpfyartebe/custom_metadata.yaml --release-name tensorflow-feedstock-15-9e99a628148d14401ddd285aeea9da4ec4096455 tensorflow-feedstock/recipe', with error: 'INFO:root:Found feedstock tensorflow-feedstock at /tmp/tmpfyartebe/tensorflow-feedstock/recipe
INFO:root:Build/Test graph from tensorflow-feedstock
Created by sbx_user1051 on 2026-01-09 16:50:34
Command line: /opt/conda/envs/webhook-listener/bin/pbp generate from-feedstock --output /tmp/tmpfyartebe/tensorflow-feedstock.yaml --custom-metadata-file /tmp/tmpfyartebe/custom_metadata.yaml --release-name tensorflow-feedstock-15-9e99a628148d14401ddd285aeea9da4ec4096455 tensorflow-feedstock/recipe

INFO:root:Found abs.yaml in /tmp/tmpfyartebe/tensorflow-feedstock
INFO:root:Creating graph from feedstock at: tensorflow-feedstock
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = linux-64.
INFO:pbpcli.utils.expand_variants:Elapsed time for get_package_variants: 0.10 seconds
INFO:pbpcli.utils.expand_variants:Matrix vars: 4 unchanging vars:(456), changing vars (4): ['blas_impl', 'cuda_compiler_version', 'gpu_variant', 'python']
INFO:pbpcli.utils.expand_variants:Elapsed time for preparation for render: 0.00 seconds
WARNING: Adding .* to spec 'protobuf 6.33.0' to ensure satisfiability.  Please consider putting {{{{ var_name }}}}.* or some relational operator (>/</>=/<=) on this spec in meta.yaml, or if req is also a build req, using {{{{ pin_compatible() }}}} jinja2 function instead.  See https://conda.io/docs/user-guide/tasks/build-packages/variants.html#pinning-at-the-variant-level
WARNING: Adding .* to spec 'python-flatbuffers 24.3.25' to ensure satisfiability.  Please consider putting {{{{ var_name }}}}.* or some relational operator (>/</>=/<=) on this spec in meta.yaml, or if req is also a build req, using {{{{ pin_compatible() }}}} jinja2 function instead.  See https://conda.io/docs/user-guide/tasks/build-packages/variants.html#pinning-at-the-variant-level
INFO:pbpcli.utils.expand_variants:Elapsed time for api.render: 91.06 seconds
INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = linux-aarch64.
INFO:pbpcli.utils.expand_variants:Elapsed time for get_package_variants: 0.10 seconds
INFO:pbpcli.utils.expand_variants:Matrix vars: 1 unchanging vars:(458), changing vars (1): ['python']
INFO:pbpcli.utils.expand_variants:Elapsed time for preparation for render: 0.00 seconds
INFO:pbpcli.utils.common:Setting env var: CONDA_OVERRIDE_GLIBC = 2.28.
INFO:pbpcli.utils.common:Unsetting env var: CONDA_OVERRIDE_GLIBC
INFO:pbpcli.utils.expand_variants:Elapsed time for api.render: 64.44 seconds
INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = osx-arm64.
INFO:pbpcli.utils.expand_variants:Elapsed time for get_package_variants: 0.10 seconds
INFO:pbpcli.utils.expand_variants:Matrix vars: 1 unchanging vars:(462), changing vars (1): ['python']
INFO:pbpcli.utils.expand_variants:Elapsed time for preparation for render: 0.00 seconds
INFO:pbpcli.utils.common:Setting env var: CONDA_OVERRIDE_OSX = 12.1.
WARNING: Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
INFO:pbpcli.utils.common:Unsetting env var: CONDA_OVERRIDE_OSX
INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/bin/pbp", line 10, in <module>
    sys.exit(cli())
             ~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1442, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1363, in main
    rv = self.invoke(ctx)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1226, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 794, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 413, in wrapper
    return func(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 624, in from_feedstock
    build_configs, test_configs = expand_build_matrix(feedstock.recipe_dir_path, src_channels)
                                  ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 79, in expand_build_matrix
    _targets, build_configs, test_configs = variants_for_feedstock_and_platform(
                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        recipe_dir_path, platform, src_channels
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/utils/expand_variants.py", line 544, in variants_for_feedstock_and_platform
    metadata_tuples = api.render(
        parsed.recipe,
    ...<2 lines>...
        variants=None,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/api.py", line 68, in render
    return render_metadata_tuples(
        metadata_tuples,
    ...<3 lines>...
        bypass_env_check=bypass_env_check,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 1097, in render_metadata_tuples
    for od, om in meta.get_output_metadata_set(
                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        permit_unsatisfiable_variants=permit_unsatisfiable_variants,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        permit_undefined_jinja=not finalize,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        bypass_env_check=bypass_env_check,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ):
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 2781, in get_output_metadata_set
    conda_packages = finalize_outputs_pass(
        ref_metadata,
    ...<3 lines>...
        bypass_env_check=bypass_env_check,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 1077, in finalize_outputs_pass
    fm = finalize_metadata(
        om,
        parent_metadata=parent_metadata,
        permit_unsatisfiable_variants=permit_unsatisfiable_variants,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 722, in finalize_metadata
    build_unsat, host_unsat = add_upstream_pins(
                              ~~~~~~~~~~~~~~~~~^
        m, permit_unsatisfiable_variants, exclude_pattern, extra_specs
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 507, in add_upstream_pins
    host_deps, host_unsat, extra_run_specs_from_host = _read_upstream_pin_files(
                                                       ~~~~~~~~~~~~~~~~~~~~~~~~^
        m, "host", permit_unsatisfiable_variants, exclude_pattern, extra_specs
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 463, in _read_upstream_pin_files
    extra_run_specs = get_upstream_pins(m, precs, env)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 433, in get_upstream_pins
    loc, dist = execute_download_actions(
                ~~~~~~~~~~~~~~~~~~~~~~~~^
        m,
        ^^
    ...<2 lines>...
        package_subset=[prec],
        ^^^^^^^^^^^^^^^^^^^^^^
    )[prec]
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 396, in execute_download_actions
    pfe.execute()
    ~~~~~~~~~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda/core/package_cache_data.py", line 918, in execute
    raise CondaMultiError(not_cancelled)
conda.CondaMultiError: Error with archive /tmp/tmpfyartebe/.conda/pkgs/python-3.13.11-hc7d8306_100_cp313.conda.  You probably need to delete and re-download or re-create this file.  Message was:

failed with error: [Errno 28] No space left on device

'
aws lambda request id: 8c74caf4-4c59-5f53-bed0-ab2f784eb7f9
[View in Grafana](https://grafana.anacondaconnect.com/explore?schemaVersion=1&panes=%7B%2286g%22:%7B%22datasource%22:%22loki-central%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7B__aws_cloudwatch_log_group%3D%5C%22%2Faws%2Flambda%2Fpbp-graph-submitter-prod%5C%22%7D%20%7C%20json%20%7C%20requestId%20%3D%20%608c74caf4-4c59-5f53-bed0-ab2f784eb7f9%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki-central%22%7D,%22editorMode%22:%22builder%22,%22direction%22:%22backward%22%7D%5D,%22range%22:%7B%22from%22:%22now-3h%22,%22to%22:%22now%22%7D,%22panelsState%22:%7B%22logs%22:%7B%22columns%22:%7B%220%22:%22Time%22,%221%22:%22Line%22%7D,%22visualisationType%22:%22table%22,%22labelFieldName%22:%22labels%22%7D%7D%7D%7D&orgId=1)</details>

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,4 +1,1 @@
 task_timeout: 54000
-
-channels:
-  - https://staging.continuum.io/pbp/fs/libprotobuf-feedstock/pr35/fe27eee

